### PR TITLE
Fix an indentation style issue in TransformsIntoAircraft

### PR DESCRIPTION
**Pull request by abcdefg30**
Came across this by pure chance.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Buildings/TransformsIntoAircraft.cs [hunk 1]
```diff
@@ -63,7 +63,7 @@ IEnumerable<IOrderTargeter> IIssueOrder.Orders
 				if (!IsTraitDisabled)
 				{
 					yield return new EnterAlliedActorTargeter<BuildingInfo>("Enter", 5, AircraftCanEnter,
-					 target => Reservable.IsAvailableFor(target, self));
+						target => Reservable.IsAvailableFor(target, self));
```
Comment (reaperrr) [inline OpenRA.Mods.Common/Traits/Buildings/TransformsIntoAircraft.cs]: While you're at it, wouldn't it help readability to insert an empty line below?

Comment (abcdefg30) [reply OpenRA.Mods.Common/Traits/Buildings/TransformsIntoAircraft.cs]: Done.

**Review by Mailaender — APPROVED** [review]

**Review by teinarss — APPROVED** [review]

**Comment by teinarss** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/f8d22b24db322ac6b2c654a77c7af5f40ffbd093...4ae3877f425203bb2212b6714dbb7898fc13d475

**Comment by GraionDilach** [comment]
@teinarss You forgot to click on the actual merge button.

**Pull request merged**